scripts/receivesms

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+# the filename must match the given command in Main.kt
+source ./_base
```

Comment:
this script doesn't seem to do much?

Reply:
It's existence allows you to do
```shell
$ cd scripts
$ ./receivesms ...
```

Instead of doing `java -jar target/wgtwo.jar receivesms ...`. Only the filename is needed when calling the `./_base` file. Maybe it's not useful?